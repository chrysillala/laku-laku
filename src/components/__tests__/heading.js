// This is a very brief snapshot test, which uses react-test-renderer to render the component,
// and then generates a snapshot of it on the first run.
// It then compares future snapshots against this, which means you can quickly check for regressions.

import React from 'react'
import renderer from 'react-test-renderer'

import Heading from '../heading'

describe("Heading", () => {
  it("heading renders correctly", () => {
    const tree = renderer
      .create(<Heading headerText="Header Text" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})