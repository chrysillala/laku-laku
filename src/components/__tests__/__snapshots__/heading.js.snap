// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading heading renders correctly 1`] = `
<h1>
  Header Text
</h1>
`;
